### ISSUE TITLE
Offsets 

### ISSUE THREAD

**Issue by skaman82** [nootropicdesign/arduino-tvout-ve#3]
How to change the offsets so the overlay will start at 0px 0px coordinates of the monitor. As default the overlay area has quite some offset to left and right.

**Comment by skaman82** [comment]
To clarify:
![img](https://user-images.githubusercontent.com/11768024/44147108-f11efaa0-a092-11e8-8954-4e3397193ff5.jpg)
 I basically need to move the overlay canvas to one side

**Comment by nootropicdesign** [comment]
Yes, there are ways to adjust the position of the viewport. I wrote about it in this forum post:

[adjusting the viewport position of the video experimenter output](https://nootropicdesign.com/store/forums/topic/adjusting-the-viewport-position-of-the-video-experimenter-output/)
